packages/cli/test/functional/testUtil/compare.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const path = require('path');
+const ignore = require('ignore');
+const walkSync = require('walk-sync');
+const { isBinary } = require('istextorbinary');
+const diffChars = require('./diffChars');
+
+const _ = {};
+_.isEqual = require('lodash/isEqual');
```

Comment:
We don't seem to use `isEqual` in this file. Is this import necessary?

Reply:
Nice catch, I'll see if I can make the eslint ignore more specific as well